src/Tests/Indices/IndexSettings/IndexTemplates/GetIndexTemplate/GetIndexTemplateApiTests.cs

```diff
@@ -25,6 +27,46 @@ public class GetIndexTemplateApiTests
 		protected override Func<GetIndexTemplateDescriptor, IGetIndexTemplateRequest> Fluent => d => d
 			.Name(CallIsolatedValue);
 
+		protected override int ExpectStatusCode => 200;
+
+		protected override bool ExpectIsValid => true;
+
 		protected override GetIndexTemplateRequest Initializer => new GetIndexTemplateRequest(CallIsolatedValue);
+
+		protected override object ExpectJson => new object();
```

Comment:
According to `ExpectResponse`, this won't be `{}` but will be an object with index patterns, version, template mappings and settings.

Reply:
Can set this to `null`